src/js/pre-need/components/SupportingDocumentsDescription.jsx

```diff
@@ -1,46 +1,48 @@
+  if (isVeteran(props.formData)) {
+    desc = (
+      <p>If you have your DD214 or other separation papers, please upload a copy. This will help us figure out if you qualify for burial based on your military status and service history. If you don’t have these documents, you can still apply–and we’ll request the documents on your behalf.</p>
     );
-
-    const answer1 = (
-      <div className={answerClass}>
-        <p>
-          Documents may include the most recent discharge document (DD Form 214) showing the highest rank and valor awards and decorations, active duty service records, or active duty for a minimum of 24 continuous months for enlisted Servicemembers after September 7, 1980; for officers, after October 16, 1981, or the full period for which the person was called to active duty. If you are unable to locate copies of military records, apply anyway, as VA will attempt to obtain records necessary to make a determination.
-        </p>
-        <button type="button" onClick={this.toggleAnswer1}>Close</button>
+  } else if (isUnmarriedChild(props.formData)) {
+    desc = (
+      <div>
+        <p>If you have your sponsor’s DD214 or other separation papers, please upload a copy. This will help us figure out if you qualify for burial based on your sponsor’s military status and service history. If you don’t have these documents, you can still apply–and we’ll request the documents on your behalf.</p>
+        <p>You’ll also need to provide supporting documents, such as :</p>
+        <ul>
+          <li>Medical evidence of a disability</li>
+          <li>Start date of a disability</li>
+          <li>Age when diagnosed with disability</li>
+        </ul>
       </div>
     );
+  } else {
+    desc = (
+      <p>If you have your sponsor’s DD214 or other separation papers, please upload a copy. This will help us figure out if you qualify for burial based on your sponsor’s military status and service history. If you don’t have these documents, you can still apply–and we’ll request the documents on your behalf.</p>
+    );
+  }
 
-    const answer2 = (
-      <div className={answerClass}>
+  return (
+    <div id="supporting-documents-description">
+      <p>
+        If you have supporting documents readily available, you can upload them to help us make a determination quickly.
+      </p>
+      <AdditionalInfo triggerText="What kinds of documents should I provide?">
+        {desc}
+        <p>If you’re applying on behalf of someone else, you’ll need to provide supporting documents or an affidavit showing that you’re:</p>
+        <ul>
+          <li>The applicant’s court-appointed representative, or</li>
+          <li>The applicant’s caregiver (including a spouse or other relative), or</li>
+          <li>An attorney or agent acting on behalf of the applicant under a durable power of attorney, or</li>
+          <li>The manager or principal officer of an institution in which the applicant is being care for</li>
```

Comment:
Should this be "cared for" (with a "d")? Also, it ends with a preposition, not sure if that's something we want to try to avoid.